Features/desktop contact

Wahaj-Ali opened the pull request:
This is the final branch and it contains
- the desktop version of the header section
- the desktop version of the healine section
- the desktop version of the work/projects section
- the desktop version of the about me section
- the desktop version of the contact section

Wahaj-Ali committed:
Add files from another repo containing files for Portfolio-Mobile-version

in this commit
- add previous html and css files for Portfolio-mobile version

Wahaj-Ali committed:
Set up Desktop header

in this commit
- used media query to set up header for screen width more than 786px.
- removed hamburger icon from the header
- add anchor tags for portfolio, contact, about-me, and mail on the header
- styled header to fit accordingly

Wahaj-Ali committed:
Set up Headline section for desktop version of Portfolio

in this commit
- added media query for headline section elements
- replaced the background image for desktop version
- set the fonts accordingly

Wahaj-Ali committed:
Set up Projects section for desktop version

in this commit
- modify works/projects section for desktop version of my portfolio
- styled index.html accordingly

Wahaj-Ali committed:
Set up "about me" and "Contact" sections for Desktop version of Portfolio

in this commit
- set up about me section for desktop version of portfolio
- set up contact section for desktop version of portfolio.

Wahaj-Ali committed:
Update styles.css

in this commit
- removed linter errors

Wahaj-Ali committed:
Update styles.css

in this commit
- removed further linter errors

Wahaj-Ali committed:
Update styles.css

in this commit
- removed extra space by styling html body

Wahaj-Ali committed:
Update styles.css

in this commit
- removed linter errors

Wahaj-Ali committed:
Update styles.css

in this commit
- removed linter error

Wahaj-Ali committed:
Update styles.css

in this commit
- removed another linter error

sajid-munawar reviewed (changes requested):
Hi there, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️

   - You have submitted a project that lacks REAMDE file. Kindly add the README file in which it shows what your project is talking about.




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

>Invalid Code Review Request **does not count** into the code reviews limit.

Wahaj-Ali committed:
Update README.md

in this commit
- add descriptive README

mckent05 commented on styles.css:
```diff
@@ -0,0 +1,967 @@
+  flex: 0 10 1px;
+}
+
+.container-headline {
+  display: flex;
+  flex-wrap: wrap;
+  background-color: black;
+  flex-direction: column;
+  width: 370px;
+  background-image: url(/attributes/illustration-about-me-mobile.png);
+  background-repeat: no-repeat;
+  background-position-x: right;
+  background-position-y: 24pc;
+}
```
- The background image in this section is not displaying correctly, this is likely because you aren't navigating into the attributes folder correctly. Your URL path is missing a `.` i.e `./atrributes/...`. Please fix this 👍 👍 
- Please fix this for all images or background images as it may apply. 

Your project:
<img width="391" alt="project" src="https://user-images.githubusercontent.com/73607512/199681542-bbd481d7-ba80-444d-a42a-a1b8648eb02a.PNG">

Figma template:
<img width="497" alt="2" src="https://user-images.githubusercontent.com/73607512/199681636-b02fc659-cdc9-4843-a096-31b1c38c4d72.PNG">

Wahaj-Ali replied:
![Screenshot 2022-11-03 155503](https://user-images.githubusercontent.com/111431787/199703981-c1921919-6335-409c-89f6-d10c1602bd31.png)
@mckent05 There must be an issue while you run my code. It is working completely fine when I run it on my machine.

mckent05 commented on index.html:
```diff
@@ -0,0 +1,304 @@
+                    <li><a href=""><img id="en_btn" src="/attributes/En.png" alt="En" /></a></li>
+                </ul>
+    </section>
+    <section class="container-works">
+        <h3 id="works_heading">My Recent Works</h3>
+        <hr class="solid">
+        <section class="projects">
+            <img class="works_image" src="/attributes/Img Placeholder@3x.png" />
+            <h4 id="multi_heading">Multi-post Stories</h4>
+            <p id="works_intro">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+                been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard
+                dummy text.</p>
+            <article class="lang-container" >
+                <ul style="list-style-type: none;" class="list1">
+                <li class="lang">CSS</li>
+                <li class="lang">HTML</li>
+                <li class="lang">Bootstrap</li>
+                <li class="lang">Ruby</li>
+            </ul>
+            </article>
+            <button type="submit" class="checkout-button" id="projects-btn1" name="checkout-button"
+                enable="disabled">See Project</button>
+        </section>
+    </section>
```
- This section of your page has a few differences from what is given in the design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏. Please fix this 👍 👍 

Your project:
-- The section divider is overflowing into the `My Recent Projects` text. 
-- The project button card is not aligned as it is in the Figma template. Please fix this. 😄 
-- The image on the project card is not displaying (due to the reason I mentioned below). Please fix this 😄
-- Some of the text on the right side of your page are hidden and not displaying. Please fix this 👍 👍 
-- The background for this section should be white, please fix this 🙏 
<img width="947" alt="qw" src="https://user-images.githubusercontent.com/73607512/199683647-690b7b93-ee37-4402-81e5-8ae4bd42b509.PNG">

Mobile version:
<img width="435" alt="qa" src="https://user-images.githubusercontent.com/73607512/199692531-67b4a275-edb7-42a1-bf75-3949cb27cc0e.PNG">


Figma template:
<img width="471" alt="3" src="https://user-images.githubusercontent.com/73607512/199683941-b70556b8-3ed8-4d96-b855-aee6560c9404.PNG">

Wahaj-Ali replied:
![Screenshot 2022-11-03 155521](https://user-images.githubusercontent.com/111431787/199704226-d28126c7-f093-4b73-9a3e-639bb84029e5.png)
@mckent05 Same case here as well.

Wahaj-Ali replied:
@mckent05 you might not have downloaded the complete project. It contains all the required files and it is working fine.

mckent05 commented on index.html:
```diff
@@ -0,0 +1,304 @@
+                    </ul>
+                </article>
+        </section>
+    </section>
```
- To ensure that your HTML is clean and easy to maintain, I do not advise that you have 2 different sections for your project cards (i.e desktop version and mobile version). Please have just one section on your HTML page and use media-query in CSS to style this section for either mobile or desktop. Please try to fix this 🙏

mckent05 commented on index.html:
```diff
@@ -0,0 +1,304 @@
+        </article>
+        <button type="submit" class="checkout-button" id="projects-btn2" name="checkout-button" enable="disabled">See
+            Project</button>
+    </section>
```
- Relating to my review above, I suggest that you use this section to display your work section for both desktop and mobile screen sizes, as it seems more semantically correct. Please delete the `desktop-project-section` above and use media-queries to align this section for both mobile and desktop screen sizes. 👍 👍 
- Also, I suggest that you wrap the whole of this section in a `section` tag and use the `grid` display property in CSS to align the project cards. Please fix this 👍 👍

mckent05 commented on styles.css:
```diff
@@ -0,0 +1,967 @@
+  margin-top: 14px;
+}
+
+.about_me {
+  background-color: black;
+  width: 370px;
+  height: 1695px;
+  background-image: url(/attributes/illustration-about-me-mobile.png);
+  background-repeat: no-repeat;
+  background-position-x: right;
+  background-position-y: 24pc;
+  display: flex;
+  flex-direction: column;
+}
```
- The background image in this section is not displaying correctly, this is likely because you aren't navigating into the attributes folder correctly. Your URL path is missing a . i.e ./atrributes/.... Please fix this 👍 👍

Your project:
<img width="945" alt="qwn" src="https://user-images.githubusercontent.com/73607512/199688917-9ddf0961-0028-4270-845d-50142575764d.PNG">

Figm template:
<img width="495" alt="qqq" src="https://user-images.githubusercontent.com/73607512/199689052-b88683e5-7eb2-479a-8171-c78a62956277.PNG">

Wahaj-Ali replied:
![Screenshot 2022-11-03 155552](https://user-images.githubusercontent.com/111431787/199705061-c7ebfcb4-6cc3-43c6-b483-0add6f65fdc8.png)
@mckent05

mckent05 commented on index.html:
```diff
@@ -0,0 +1,304 @@
+        </section>
+    </section>
+
+    <section class="contact">
+        <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+        <form id="form" action="https://formspree.io/f/xayklnzr" method="post">
+            <input type="text" style="border:none" placeholder="Full Name" name="Name" id="full_name" maxlength="30" required>
+            <hr id="divider1">
+            <input type="email" style="border:none" placeholder="Email Address" name="email" id="full_name" required>
+            <hr id="divider1">
+            <textarea id="message" name="message" placeholder="Enter text here" class="fcf-form-control" rows="6" maxlength="500" required></textarea>
+            <button type="submit" class="checkout-button" id="getintouch_btn" >Get in touch</button>
+        </form>
```
- This section of your page has some differences from what is in the design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your Project:
-- There should be two input fields for the name. Please fix this 😄 
- Also, please fix the width for the form inputs 👍 👍 
-- Also, remember to implement the styling when the buttons are hovered on. Please fix this for all buttons
<img width="949" alt="smile" src="https://user-images.githubusercontent.com/73607512/199690675-f3c36a23-23b2-4af3-99d3-0d7931033cb0.PNG">

Figma Design:
<img width="524" alt="saas" src="https://user-images.githubusercontent.com/73607512/199691278-9e15da7b-3de2-4f74-9638-a9dfa72af590.PNG">



Hover design:
<img width="369" alt="aq" src="https://user-images.githubusercontent.com/73607512/199690893-00ac158b-bfd0-43cc-b0e0-8ec3db00241f.PNG">

Wahaj-Ali replied:
![Screenshot 2022-11-03 155623](https://user-images.githubusercontent.com/111431787/199705283-260678fc-1589-481b-bcb8-cbd499627c37.png)
@mckent05 I have used the hover property on the "Get in touch" button.

Wahaj-Ali replied:
@mckent05 and we are not supposed to use 2 separate fields for name, instead, we were asked to use a single input for "Full Name" in the project requirements.
![Screenshot 2022-11-03 160823](https://user-images.githubusercontent.com/111431787/199705922-16f43c5e-bed1-4abb-94fe-e0fbc894b6d4.png)

Wahaj-Ali replied:
@mckent05 I think you forgot to read the instructions for this project.

mckent05 reviewed (changes requested):
Hi @Wahaj-Ali, 

# STATUS: CHAN GES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![almost there](https://user-images.githubusercontent.com/73607512/199692644-e45fbe1e-0caf-46f5-8d8b-0bb273330eee.gif)

## Highlights
- All linter checks are passing ✔️ 
- Descriptive commit messages 👏 👏 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Lameck1 commented on index.html:
```diff
@@ -0,0 +1,304 @@
+            been the industry's standard</p>
+        <article class="lang_container_2">
+            <ul style="list-style-type:none;" class="list">
+                <li id="lang1">HTML</li>
+                <li id="lang2">Bootstrap</li>
+                <li id="lang3">Ruby</li>
+            </ul>
+        </article>
+        <button type="submit" class="checkout-button" id="projects-btn2" name="checkout-button" enable="disabled">See
+            Project</button>
+    </section>
+    <section class="works">
+        <h3 id="wors_descp">Professional Art Printing Data</h3>
+        <p id="work_detail">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+            been the industry's standard</p>
+        <article class="lang_container_2">
+            <ul style="list-style-type:none;" class="list">
+                <li id="lang1">HTML</li>
+                <li id="lang2">Bootstrap</li>
+                <li id="lang3">Ruby</li>
+            </ul>
+        </article>
+        <button type="submit" class="checkout-button" id="projects-btn2" name="checkout-button" enable="disabled">See
+            Project</button>
+    </section>
+    <section class="works">
+        <h3 id="wors_descp">Professional Art Printing Data</h3>
+        <p id="work_detail">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+            been the industry's standard</p>
+        <article class="lang_container_2">
+            <ul style="list-style-type:none;" class="list">
+                <li id="lang1">HTML</li>
+                <li id="lang2">Bootstrap</li>
+                <li id="lang3">Ruby</li>
+            </ul>
+        </article>
+        <button type="submit" class="checkout-button" id="projects-btn2" name="checkout-button" enable="disabled">See
+            Project</button>
+    </section>
+    <!-- mobile projects section end  -->
```
- [ ] Just like @mckent05 has pointed out, what you have done here could not be considered right or best practice. In responsive design, we need to work with the same content but use media queries to lay the content responsively for smaller and larger screens. Please follow the suggestion that @mckent05 left above.

Lameck1 reviewed (commented):
Hello, @Wahaj-Ali 👋,

_This is an additional review as a result of the second opinion request._

I can confirm that the app works as expected. However, you'll need to address the issue highlighted in the inline comment below before this project can be considered complete. Please follow the suggestion that @mckent05 left for you in his review concerning this.

Keep rocking 🚀!

Wahaj-Ali committed:
Removed unnecessary code from index html as required by the reviewer

in this commit
- used single code for both mobile and desktop versions by removing code from index.html and using media query for the remaining code.

Wahaj-Ali committed:
Update styles.css

in this commit
- removed linter errors

Wahaj-Ali committed:
Update styles.css

-removed linter error for an extra line at the end.

TuyishimireEric commented on index.html:
```diff
@@ -0,0 +1,227 @@
+            <h3 class="project_descp1">Professional Art Printing Data</h3>
+            <p class="project_detail">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+                been the industry's standard</p>
+                <article class="lang_container_2">
+                    <ul style="list-style-type:none;" class="list">
+                        <li id="lang1">HTML</li>
+                        <li id="lang2">Bootstrap</li>
+                        <li id="lang3">Ruby</li>
+                    </ul>
+                </article>
+                <button type="submit" class="checkout-button" id="projects-btn2" name="checkout-button" enable="disabled">See
+                    Project</button>
+        </section>
+        <section class="project" id="prcjt2">
+            <h3 class="project_descp1">Data Dashboard Healthcare</h3>
+            <p class="project_detail">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+                been the industry's standard</p>
+                <article class="lang_container_2">
+                    <ul style="list-style-type:none;" class="list">
+                        <li id="lang1">HTML</li>
+                        <li id="lang2">Bootstrap</li>
+                        <li id="lang3">Ruby</li>
+                    </ul>
+                </article>
+                <button type="submit" class="checkout-button" id="projects-btn2" name="checkout-button" enable="disabled">See
+                    Project</button>
+        </section>
+        <section class="project" id="prcjt3">
+            <h3 class="project_descp1">Data Dashboard Healthcare</h3>
+            <p class="project_detail">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+                been the industry's standard</p>
+                <article class="lang_container_2">
+                    <ul style="list-style-type:none;" class="list">
+                        <li id="lang1">HTML</li>
+                        <li id="lang2">Bootstrap</li>
+                        <li id="lang3">Ruby</li>
+                    </ul>
+                </article>
+                <button type="submit" class="checkout-button" id="projects-btn2" name="checkout-button" enable="disabled">See
+                    Project</button>
```
- [ ] **As suggested by the previous reviewers,**  Just to ensure that your HTML is clean and easy to maintain, It would be better to have **2 different sections** for your project cards (i.e desktop version and mobile version). :joy: Kindly fix this to make your code cleaner :+1: :+1: 

    I think it would be better to replace some of the ```section``` tag by ```div``` tag as the ```div``` is an empty container that generally defines a division type or section type; it does not affect the contents whatever the developer writes the front end code or formats like headers, footers or any layout changes also both horizontal and vertical layout, it does not affect it. :joy: 

you may follow this l[ink](https://www.educba.com/html-section-vs-div/) for difference between ```section``` and ```div``` tags

Wahaj-Ali replied:
@TuyishimireEric You are mistaken. Kindly check the code again. There is only one section for both mobile and desktop versions. Please check carefully!!!

Wahaj-Ali replied:
@TuyishimireEric It's a request to kindly check our projects carefully as we spend a lot of time making them.

Wahaj-Ali replied:
@TuyishimireEric if you carefully read the above comments by previous reviewers carefully, you will know that I made the correct required changes.

TuyishimireEric reviewed (changes requested):
# Status: REQUIRED CHANGES :recycle: 
### Hi  @Wahaj-Ali   👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job so far! 🥇 
There are some issues that you still need to work on to go to the next project but you are almost there!
![](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966)

## Highlights 
:heavy_check_mark: No Linter Errors :+1: 
:heavy_check_mark: Professional PR :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] [OPTIONAL] I think it would be better to remember to add always the quotations in your ```background-image: url(" ")``` to make sure that your pictures will be visible in all browsers, :joy: 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

bimbolabuari reviewed (commented):
Hello @Wahaj-Ali  👋,

_This is an additional review as a result of the second opinion request._

- Nice implementation of the requested changes 👍 
 
    - @TuyishimireEric, it is not a good practice to have different sections for different screen sizes, however, as pointed out by the first reviewer, it is only suitable to use media queries to change the section layout in order to be responsive. Let's remember to understand the previously requested changes before requesting changes 👍 
    - For consistency's sake, any requested change(s) that was not pointed out by the previous CR should be marked as optional using the optional tag.
 While it is semantically correct to use a `div` or `ul` tag to divide each project card, this was not mentioned by the previous CR  and should not stop the project approval. 👍 

- @Wahaj-Ali, I strongly advise you to take up the CR suggestion on the usage of `section` and `div` tags, however, it does not stop the project approval. 👍 

As this was the only reason your project was not approved it will be approved as soon as possible.

Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approved right away :)

Keep rocking 🚀!

jerryowusu commented on index.html:
```diff
@@ -0,0 +1,227 @@
+                    Project</button>
+        </section>
+    </section>
+    <section class="about_me">
+        <h2 id="about_text">About Me</h2>
+        <p id="intro_aboutme">Hello I’m a software developer! I can help you build a product , feature or website Look
+            through some of my work and experience! If you like what you see and have a project you need coded, don’t
+            hestiate to contact me.</p>
+        <button type="submit" class="checkout-button" id="resume-btn" name="checkout-button" enable="disabled">Get My
+            Resume</button>
+        <hr id="divider">
+        <section id="lang_sec">
+            <img id="rect" src="/attributes/Rectangle 55.png" />
+            <p class="lang_use">Language</p>
+        </section>
+
+        <section class="artc_lang">
+            <ul style="list-style-type:none;">
+                <li class="lang_btn" id="langbtn1">Javascript</li>
+                <li class="lang_btn" id="langbtn2">Ruby</li>
+                <li class="lang_btn" id="langbtn3">HTML</li>
+                <li class="lang_btn" id="langbtn4">CSS</li>
+            </ul>
+        </section>
+
+        <section id="fram_sec">
+            <img id="sqr" src="/attributes/Rectangle 55.png" />
+            <p class="lang_use">Frameworks</p>
+        </section>
+        <section class="artc_lang" id="framework_sec">
+            <ul style="list-style-type:none;">
+                <li class="lang_btn" id="frambtn1">Bootstrap</li>
+                <li class="lang_btn" id="frambtn2">Ruby on Rails</li>
+                <li class="lang_btn" id="frambtn3">RSPec</li>
+                <li class="lang_btn" id="frambtn4">CapyBara</li>
+                <li class="lang_btn" id="frambtn5">Selenium</li>
+            </ul>
+        </section>
```
[OPTIONAL] Per the layout of the figma design, this project is suppose to have the `top navigation`, `header/headline`, `projects section`, `about myself section` and `the footer(contact me)`. Semantically and for best practices, you can use a nav tag for the top navigation, header tag for the header/headline, section tags for both the projects and about myself sections, and  footer tag for the contact me. :1234: 
There is no need to sub divide the sections into subsections using the section tags as has been done here. Kindly use the article tag or div tag for all other parts which are not sections. This will help you to achieve the best optimum solution semantically as well as help your project to be more proffessional. :+1:

jerryowusu commented on styles.css:
```diff
@@ -0,0 +1,1011 @@
+  position: relative;
+}
+
+@media only screen and (min-width: 769px) {
+  body {
+    background-color: #e5e5e5;
+    margin: 0%;
+  }
+
+  .container-header {
+    width: 100%;
+    height: 72px;
+  }
+
+  .union {
+    display: none;
+  }
+
+  .desktop_headicons {
+    display: flex;
+    margin-left: auto;
+    margin-right: 150px;
+    gap: 25px;
+    align-items: center;
+    color: #fff;
+    font-family: 'Inter', sans-serif;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
```
[OPTIONAL] As you are trying to build a responsive website, you should ensure your website looks as like the figma design at all screen width and media break points as much as possible. You can consider adding more media breakpoints if it is difficult to achieve this with only one. :pray: 
Certain parts of your website does not look responsive for screen widths between 768px to 1150px as shown in the images below. Kindly check and rectify accordingly to help make your website more responsive. 

 Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-04 07-57-44](https://user-images.githubusercontent.com/65233860/199926674-a39c3a60-ef85-4154-bd55-cd22af521492.png)  |  ![Screenshot from 2022-11-04 08-16-28](https://user-images.githubusercontent.com/65233860/199926737-ecf324c3-6b68-4513-b7e8-673ed48e823d.png)

 Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-04 07-58-02](https://user-images.githubusercontent.com/65233860/199926954-2ac3cb4d-0c6a-4685-8eea-1741220228be.png)  |  ![Screenshot from 2022-11-04 08-32-02](https://user-images.githubusercontent.com/65233860/199928085-4a5380ec-c556-4e03-81e9-38688dec49a1.png)


 Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-04 08-29-44](https://user-images.githubusercontent.com/65233860/199927564-a9b364c3-a2f8-4ab3-875a-6f7d16d13e96.png)  |  ![Screenshot from 2022-11-04 08-16-51](https://user-images.githubusercontent.com/65233860/199927605-c60c58e7-917c-4a57-a814-28e055e20d9e.png)

jerryowusu reviewed (approved):
Hi @Wahaj-Ali 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![200w](https://user-images.githubusercontent.com/65233860/199928809-e0b08ec7-a28c-4aeb-8e43-36da5527a7a3.gif)

## Highlights
Excellent communication with reviewers :heavy_check_mark: 
Good commit messages :heavy_check_mark: 
Followed GitHub flow :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
